src/components/home/FeaturedPackages.vue

```diff
@@ -15,68 +15,33 @@
 
 <script>
   import Card from "../Card";
+  import axios from "axios";
 
   export default {
     name: "FeaturedPackages",
     data() {
       return {
-        items: [
-          {
-            packageTitle: "Hello World",
-            packageURL: "https://nest.land/x/bruh",
-            packageInfo: {
-              author: "Nanny McFee",
-              docsLink: "https://github.com/nannymcfee/helloworld",
-              description: "A simple, easy to use hello world package!",
-              version: "1.0.1",
-              publishedDate: "datetime object here"
-            }
-          },
-          {
-            packageTitle: "Cheese",
-            packageURL: "https://nest.land/x/bruh",
-            packageInfo: {
-              author: "Zorbyte",
-              docsLink: "https://github.com/nannymcfee/helloworld",
-              description: "A simple, easy to use hello world package!",
-              version: "1.0.1",
-              publishedDate: "datetime object here"
-            }
-          },
-          {
-            packageTitle: "a1",
-            packageURL: "https://nest.land/x/bruh",
-            packageInfo: {
-              author: "sauce",
-              docsLink: "https://github.com/nannymcfee/helloworld",
-              description: "A simple, easy to use hello world package!",
-              version: "1.0.1",
-              publishedDate: "datetime object here"
-            }
-          },
-          {
-            packageTitle: "yo",
-            packageURL: "https://nest.land/x/bruh",
-            packageInfo: {
-              author: "mama",
-              docsLink: "https://github.com/nannymcfee/helloworld",
-              description: "A simple, easy to use hello world package!",
-              version: "1.0.1",
-              publishedDate: "datetime object here"
-            }
-          }
-        ]
+        featuredModules: [],
       };
     },
     components: {
       Card
-    }
+    },
+    async created () {
+      //selecting 4 random packages from the last 20 updated packages
+      await axios
+        .get("https://x.nest.land/api/packages/20")
+        .then(response => {
+          this.featuredModules = response.data.sort(() => 0.5 - Math.random()).slice(0, 4);
```

Comment:
1. This requests the last 20 updated modules
2. Shuffles the array of modules
3. Selects the first 4 modules to feature

Reply:
Oh wow that is really neat! I wasn't thinking it would work like that, but that'll work! Super cool.